Moved some stuff around, made index/about in ITA and ENG (#19)
nientedidecente/nientedidecente.github.io: en/about.md

---
layout: page
title: About
lang: en
lang-ref: about-page
---

## Who are we?

We are just a bunch of guys who replied to a reddit post in [here](https://www.reddit.com/r/ItalyInformatica/comments/giby9r/community_italiana_game_developer_open_source_ii/)(ITA).

> Different ages and different professional competencies.

Reviewer: ```suggestion
We're people from various backgrounds and different ages.
```

> We are **coders**, **musicians**, **students**, **professional**, but more importantly, gamers.

Reviewer: ```suggestion
We are **coders**, **musicians**, **students**, **professionals**, but most importantly, gamers.
```

More info in [here](./team).

## What do we do?

We are still working on it, at the moment we have setup:

> - A [Discord](https://discord.gg/ysJUrSm) server, where we all hangout most of the time.

Reviewer: ```suggestion
- A [Discord](https://discord.gg/ysJUrSm) server, where we all hang out most of the time.
```

- A [GitHub](https://github.com/nientedidecente) org, where we plan to collaborate over gamedev projects.

> - This blog made with github pages.

Reviewer: ```suggestion
- This blog made with GitHub Pages.
```

## What do we want?

> The aim of any community is, obviously to hangout and collaborate with people with same interests.

Reviewer: ```suggestion
The aim of any community is obviously to hang out and collaborate with people with same interests.
```

> Our community is no different, and aside the basic, we have the main aim of creating a small virtual hub to get together people who wants to create videogames or people who wants to learn how to do one collaborating on existing projects.

Reviewer: ```suggestion
Our community is no different, and aside from the basics, our main aim is creating a small virtual hub to get together people who want to create videogames, or learn how to by collaborating on existing projects.
```
